pkg/log/log_messages.go

```diff
@@ -79,4 +79,18 @@ const CAKC063 string = "CAKC063 Unable to find configuration for URL: %s"
 const CAKC064 string = "CAKC064 Unable to find Authenticator for this configuration"
 const CAKC065 string = "CAKC065 File path is invalid %s"
 const CAKC066 string = "CAKC066 Performing authn-jwt"
-const CAKC067 string = "CAKC067 Failed reading jwt token from %s"
+const CAKC067 string = "CAKC067 Failed reading JWT from %s"
+const CAKC068 string = "CAKC068 Loading configuration object..."
+const CAKC069 string = "CAKC069 Choosing configuration object...."
+const CAKC070 string = "CAKC070 Chosen %q configuration"
+const CAKC071 string = "CAKC071 Gathering input settings for configuration..."
+const CAKC072 string = "CAKC072 Successfully gathered input settings"
+const CAKC073 string = "CAKC073 Validating input configuration..."
+const CAKC074 string = "CAKC074 Validated input configuration successfully"
+const CAKC075 string = "CAKC075 Chosen %q Authenticator"
+const CAKC076 string = "CAKC076 Loading JWT token from %s..."
+const CAKC077 string = "CAKC077 Loaded JWT token successfully"
+const CAKC078 string = "CAKC078 Extracting application identity (host) from the configuration..."
+const CAKC079 string = "CAKC079 Extracted identity %s from configuration"
```

Comment:
"Extracted identity" - is this the application identity?
```suggestion
const CAKC079 string = "CAKC079 Extracted application identity (host) %s from configuration"
```

Reply:
This is the host being authenticated in the authentication request. The host represents identity of the application